### DISCUSSION TITLE
Possible tools/utilities to help convert/implement this project into/with other languages.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [goodluck1982/SpaceGroupIrep#22] [General]
Here are some possible tools/utilities to help convert/implement this project into/with other languages:

1. [Converting Mathematica to Python](https://juanjose.garciaripoll.com/blog/converting-mathematica-to-python/
).
2. [An open-source Mathematica](https://github.com/Mathics3/mathics-core).

If you find other similar tools/utilities, they can be added to the list.

Regards,
HZ

**Comment by goodluck1982** [comment]
> Here are some possible tools/utilities to help convert/implement this project into/with other languages:
> 
> 1. [Converting Mathematica to Python](https://juanjose.garciaripoll.com/blog/converting-mathematica-to-python/).
> 2. [An open-source Mathematica](https://github.com/Mathics3/mathics-core).

1. This tool only converts an expression to python and is far from adequate to convert the whole project to python.
2. This tool is interesting. It may help. However, the main difficulty still lies in the lack of manpower. At present I have no time to do this on my own.
 
Thanks anyway for your information.

**Comment by hongyi-zhao** [comment]
Let it be and don't force it. I filed this issue also to remind myself of some relevant ideas and tools which can serve some degree of manpower savings.
